File: src/main/java/pl/com/dolittle/mkelo/entity/Result.java
package pl.com.dolittle.mkelo.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Result implements Serializable {

    private Player player;
    private int eloBefore;
    private int eloAfter;
}

File: mleko/src/app/model/Result.ts
import {Player} from "./Player";

export class Result {
  private readonly _player:Player;
  private readonly _eloBefore:number;
  private readonly _eloAfter:number;

  constructor(player: Player, eloBefore: number, eloAfter: number) {
    this._player = player;
    this._eloBefore = eloBefore;
    this._eloAfter = eloAfter;
  }

  get player(): Player {
    return this._player;
  }

  get eloBefore(): number {
    return this._eloBefore;
  }

  get eloAfter(): number {
    return this._eloAfter;
  }
}
